src/org/sosy_lab/java_smt/solvers/mathsat5/Mathsat5AbstractNativeApiTest.java

```diff
@@ -119,4 +125,48 @@ public void modularCongruence()
     assertThat(msat_check_sat(env)).isFalse(); // 4 != 45 mod 42
     msat_pop_backtrack_point(env);
   }
+
+  // msat_to_smtlib2() can not export quantified formulas, use msat_to_smtlib2_ext() instead
+  @Test
+  public void quantifierToSmtlib2() {
+    String expectedSMTLib2Formula = "Bool (forall ((x Int)) (.def_14 x))";
```

Comment:
is this valid SMTLIB? The "Bool" at start looks strange.

Reply:
I check against parts of the output. The Bool is the return type of the formula here.

I added the full output as a comment + reasoning why i don't simply check against the full output.